lib/cluster.js

```diff
@@ -350,7 +350,7 @@ function masterInit() {
     }
   }
 
-  cluster.fork = function(env) {
+  cluster.fork = (env) => {
```

Comment:
Switch a normal function to an arrow function here would be a semver-major change here. There are subtle differences here that could cause breakage.